connectors/agentic-ai/element-templates/agenticai-aiagent-job-worker.json

```diff
@@ -0,0 +1,1472 @@
+{
```

Comment:
Note: this does not work yet as the modeler throws an error on the 2 `output` properties binding to `zeebe:adHoc`.